src/Sylius/Component/Core/Model/ChannelPricing.php

```diff
@@ -30,6 +30,9 @@ class ChannelPricing implements ChannelPricingInterface
     /** @var int|null */
     protected $originalPrice;
 
+    /** @var array */
+    protected $appliedPromotions = [];
```

Comment:
Nothing to change right now, but just a question for the future - do we plan to have object references here?

Reply:
I don't know yet - we will see what needs we will have when the feature would be growing. But I do not rule it out 🖖